now.json

```diff
@@ -2,12 +2,7 @@
   "headers": [
     {
       "source": "/(.*)",
-      "headers" : [
-        {
-          "key" : "X-Robots-Tag",
-          "value" : ""
-        }
-      ]
+      "headers": [{ "key": "X-Robots-Tag", "value": "" }]
```

Comment:
This doesn't follow the normal prettified JSON format.

Reply:
I think it's more readable and `prettier` is fine with it.